src/app/modules/core/pipes/string-filter.pipe.ts

```diff
@@ -0,0 +1,24 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'stringFilterBy'
+})
+export class StringFilterByPipe implements PipeTransform {
+
+  transform(arr: any[], searchText: string,fieldName?:string): any[] {
```

Comment:
Can you explain the use of filter here?

Reply:
this filter is used in State and City dropdown